Show two months in singleDate mode

scarballo opened the pull request:
Allow show the view of two months when the singleDate option is selected. Thus avoiding extra clicks to select a day for the end user. For example, It's 31th March and you need select a day in the next month. For achieve this you need 2 clicks showing one month. With this modification only one click is necessary.

holtkamp commented:
can you provide a demo and documentation in the index.html file, and perhaps add a screenshot to this PR so everything is as clear as possible...?

scarballo commented:
@holtkamp , I've found a bug and I've fixed it in jquery.daterangepicker.js (comments inside the commit). I've updated demo.js and index.html as you requested to provide information and a demo about the new funcionality.

Here you have a working example: https://jsfiddle.net/honrrqmn/1/

Waiting for your comments.

Regards.

holtkamp commented:
@scarballo thanks for the work and proper preparation :smile: will tag a new minor version for this

holtkamp commented:
@scarballo well, it seems a minor regression has been introduced. 


When in singleDate mode, two months will be displayed by default (if screen width allows it). However, now also the time is displayed by default:

**before: 0.1.2**

<img width="396" alt="screen shot 2016-04-01 at 11 23 03" src="https://cloud.githubusercontent.com/assets/776405/14203102/30a72f40-f7fc-11e5-93b4-7e7e6e6b5b26.png">

**after: 0.2.0**
<img width="468" alt="screen shot 2016-04-01 at 11 19 25" src="https://cloud.githubusercontent.com/assets/776405/14203066/07d44e7c-f7fc-11e5-95f8-b0ca6c7defec.png">

You got any idea how this is possible and perhaps come up with a fix? :smile:

scarballo commented:
@holtkamp, Yes of course! what settings are you using? Could you copy them?

holtkamp commented:
oeps :blush: my fault. It seems I enabled the time by default. However in the singleDate mode it was never displayed because of this bug: https://github.com/longbill/jquery-date-range-picker/issues/170

Nevermind and thanks for the swift response!

scarballo commented:
@holtkamp no problem! I was testing all again and all was working well and I was becoming crazy looking for the bug hahah. Do you know when the new version will be available? The NPM package will be updated too? 

Thanks a lot!

holtkamp commented:
Just released: https://github.com/longbill/jquery-date-range-picker/releases

The NPM package, not sure whether it automatically detects this... Bower works...

scarballo commented:
Ok thank you! I think that the npm package has to be updated manually for christophior.
